game-frontend/src/components/user-script/ErrorLog.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FC, useState, useEffect } from 'react';
+import { Hook, Console, Decode } from 'console-feed';
+import { Message } from 'console-feed/lib/definitions/Console';
+
+const ErrorLog: FC = () => {
+  const [logs, setLogs] = useState<Message[]>([]);
+
+  useEffect(() => {
+    Hook(window.console, log => {
+      setLogs([...logs, Decode(log)]);
+    });
+  }, [logs]);
```

Comment:
This results in a new `Hook` listener upon each log change and will lead to severe performance degradations.